Backend/dev/migrations/rider-app/1117-add-column-to-merchant_service_usage_config.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE atlas_app.merchant_service_usage_config ADD COLUMN get_distances_o_s_r_m text;
+UPDATE atlas_app.merchant_service_usage_config SET get_distances_o_s_r_m ='OSRM';
```

Comment:
Hm why did you add this field?
If it will be always OSRM, then no need to add this field
If it may be changed, then name is not proper, can be something like `get_distances_for_cancel_ride`

Reply:
Pushed new changes.